packages/story-editor/src/components/publishModal/content/storyPreview.js

```diff
@@ -189,27 +181,74 @@ const Publisher = styled(Text).attrs({
   color: ${({ theme }) => theme.colors.fg.secondary};
 `;
 
-// TODO https://github.com/GoogleForCreators/web-stories-wp/issues/10584
-const ENABLE_EDIT_FEATURED_MEDIA = false;
-
```

Comment:
nit: could we remove this separately to keep PRs clean in case we need a rollback?